GitUI/Infrastructure/Telemetry/AppStartDiagnosticsReporter.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using GitCommands;
+
+namespace GitUI.Infrastructure.Telemetry
+{
+    public sealed class AppStartDiagnosticsReporter
+    {
+        private readonly ISshPathLocator _sshPathLocator = new SshPathLocator();
+
+        public void Report()
+        {
+            string sshClient;
+            var sshPath = _sshPathLocator.Find(AppSettings.GitBinDir);
+            if (string.IsNullOrEmpty(sshPath))
+            {
+                sshClient = "OpenSSH";
+            }
+            else if (GitCommandHelpers.Plink())
+            {
+                sshClient = "PuTTY";
+            }
+            else
+            {
+                sshClient = "Other";
+            }
+
+            DiagnosticsClient.TrackEvent("AppStart",
+                new Dictionary<string, string>
+                {
+                    { "Portable", AppSettings.IsPortable().ToString() },
+                    { "Git", GitVersion.Current.ToString() },
+                    { "SSH", sshClient },
```

Comment:
Setting to sshPath will be better I believe, then we can find internal/ external PuTTY as well as use of specific Other (something to improve for Other?)

Reply:
I'm ok with it.
However would we run a risk of inadvertently exposing sensitive paths?

Reply:
Still prefer the ssh path

Reply:
Ok, I don't have a strong preference

Reply:
Done
![image](https://user-images.githubusercontent.com/4403806/60593937-24af8680-9de7-11e9-86bd-8affc67de063.png)